src/types/shared-components/CRUDModalTemplate/interface.ts

```
@@ -300,3 +300,47 @@ export interface InterfaceRecurringEventProps {
+  /** Opens the modal */
+  open: () => void;
+  /** Closes the modal */
+  close: () => void;
+  /** Toggles the modal open/close state */
+  toggle: () => void;
+}
+
+/**
+ * Return type for useFormModal hook
+ */
+export interface InterfaceUseFormModalReturn<T>
```

ESLint

[error] Replace `T>` with `⏎··T,`

(prettier/prettier)